keyboards/smithrune/iron165r2_s/info.json

```
@@ -0,0 +1,848 @@
+{
+    "manufacturer": "SmithRune",
+    "keyboard_name": "Iron165R2-S",
+    "maintainer": "Gondolindrim",
+    "backlight": {
+        "breathing": true,
+        "breathing_period": 10,
+        "levels": 20,
+        "pin": "A6"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
```

`encoder` and `audio` are disabled by default
```suggestion
        "backlight": true,
        "bootmagic": true,
        "command": false,
        "console": false,
```